project/ParadoxSupport.scala

```diff
@@ -18,13 +18,7 @@ import scala.io.{Codec, Source}
 
 object ParadoxSupport {
   val paradoxWithCustomDirectives = Seq(
-    paradoxDirectives ++= Def.taskDyn {
```

Comment:
What kind of pattern is that weird `taskDyn { task { } }`? Is that somehow imposing some execution ordering? But seems not useful in this particular case (I've seen it used in paradox it seems to delete target folders before repopulating them with another task).

Reply:
I think that was there previously to get access to the logger, but that's no longer needed since c9f3e55c86c5a5f90bca855296c4676df6d0ba59